src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset class="form-section">
+        <legend>Personal information</legend>
+        <div class="input-lable">
+          <label for="surnameUser">Surname:</label>
+          <input
+            type="text"
+            id="surnameUser"
+            name="surname"
+            autocomplete="off"
+            required>
+        </div>
+        <div class="input-lable">
+          <label for="nameUser">Name:</label>
+          <input
+            type="text"
+            id="nameUser"
+            name="name"
+            autocomplete="off">
+        </div>
+        <div class="input-lable">
+          <label for="ageUser">How old are You?</label>
+          <input
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+            id="ageUser"
+            name="age">
+        </div>
+        <div class="input-lable">
+          <label for="birthDayUser">Full date of birth:</label>
+          <input
+            type="date"
+            id="birthDayUser"
+            name="birthDay">
+        </div>
+        <div class="input-lable">
+          <label for="acceptUser">I accept the term of the agreement  </label>
+          <input
+            type="checkbox"
+            id="acceptUser"
+            value="I accept">
+        </div>
+      </fieldset>
+      <fieldset class="form-section">
+        <legend>Registration</legend>
+        <div class="input-lable">
+          <label for="emailUser">E-mail:  </label>
+          <input
+            type="email"
+            id="emailUser"
+            name="email"
+            placeholder="email@example.com">
+        </div>
+        <div class="input-lable">
+          <label for="passwordlUser">Password:  </label>
+          <input
+            type="password"
+            id="passwordlUser"
+            name="password"
+            minlength="3"
+            maxlength="32">
+        </div>
+      </fieldset>
+
+      <fieldset class="form-section">
+        <legend>An interesting fact about you!</legend>
+        <div class="input-lable">
+          <p class="input-description">Do you love cats? </p>
+
+          <input
+            type="radio"
+            id="yesCat"
+            name="likeCats">
+          <label for="yesCat">Yes</label>
+
+          <input
+            type="radio"
+            id="noCat"
+            name="likeCats">
+          <label for="noCat">No</label>
+        </div>
+        <div class="input-lable">
+          <label for="colorUser">What is your favorite color?  </label>
+          <input
+            type="color"
+            id="colorUser"
+            name="color">
```

Comment:
```suggestion
          <input
            type="color"
            id="colorUser"
            name="color"
          >
```
fix your code style in all the places 🙏 😁